### PR TITLE
Max dev

### PR DESCRIPTION
**Pull request by maxsporer**

### CODE REVIEW

File: src/lib/components/ChallengeLink.svelte [hunk 1]
```diff
@@ -17,10 +18,10 @@
       {challengeLink}
     </div>
     <div class='flex items-center gap-x-2 mr-2'>
-      <a href={challengeLink} class='hover:text-primary-700'>
+      <a href={challengeLink} class='hover:text-primary-600'>
         <Icon icon='tabler:external-link' class='w-6 h-6'/>
       </a>
-      <div class='hover:text-primary-500 '>
```
Comment (Tsporer) [inline src/lib/components/ChallengeLink.svelte:23]: This should be 700.

File: src/lib/components/ChallengeLink.svelte [hunk 2]
```diff
@@ -29,6 +30,9 @@
           class='outline-0'
         />
       </div>
+      <button class='hover:text-primary-600' on:click={() => {navigator.share({url: challengeLink})}}>
```
Comment (Tsporer) [inline src/lib/components/ChallengeLink.svelte]: This appears on desktop but the button is not operational.

Comment (Tsporer) [reply src/lib/components/ChallengeLink.svelte]: I do like the symbol though. Nice choice.

File: src/routes/guess/+page.svelte [hunk 3]
```diff
@@ -7,12 +7,12 @@
...
-      <SearchBar />
-    </div>
-    <div class='my-2 py-3 w-full border-t-2'>
+  <div class='flex flex-col justify-between items-center'>
+    <div class='my-2 py-3 w-full border-b-2'>
       <PlayButton src={data.Item.preview.S} showTime={true}/>
     </div>
+    <div class='w-full h-[calc(100dvh-90px)] overflow-hidden'>
+      <SearchBar placeholder='Search to guess song'/>
+    </div>
```
Comment (Tsporer) [inline src/routes/guess/+page.svelte:16]: Nicely done.

**Review by Tsporer — CHANGES_REQUESTED** [review]
Good work overall 형. Please correct and we will be sugoi.

File: src/lib/components/ChallengeLink.svelte (hunk 1 above)
Comment (maxsporer) [reply src/lib/components/ChallengeLink.svelte:23]: No it should not.

File: src/routes/guess/+page.svelte (hunk 3 above)
Comment (maxsporer) [reply src/routes/guess/+page.svelte:16]: Thank you.

**Comment by Tsporer** [comment]
h

**Review by Tsporer — APPROVED** [review]

**Pull request merged**